src/Altinn.App.Api/Controllers/RedirectController.cs

```diff
@@ -16,7 +16,6 @@ namespace Altinn.App.Api.Controllers
     /// <summary>
     /// Controller for redirect and validation of URL
     /// </summary>
```

Comment:
We should add [AllowAnonymous] to leave no doubt that this Controller is open
```suggestion
    /// </summary>
    [AllowAnonymous]
```